contracts/BalanceReporters.sol

```diff
@@ -105,15 +105,17 @@ contract BalanceReporters is IBalanceReporters, ReentrancyGuardUpgradeable, Owna
         uint256 nonce = totalRewardsNonce.current();
         bytes32 candidateId = keccak256(abi.encodePacked(address(rewardEthToken), nonce, _newTotalRewards));
         bytes32 voteId = keccak256(abi.encodePacked(msg.sender, candidateId));
-        require(!submittedVotes[voteId], "BalanceReporters: rwETH total rewards vote was already submitted");
+        require(!submittedVotes[voteId], "BalanceReporters: already voted");
```

Comment:
What's the purpose of shortening error messages throughout this PR?  To save gas in the revert cases?

Reply:
It's for saving the deployment gas as it should fit into 32 bytes